samples/cpp/benchmark_app/main.cpp

```diff
@@ -189,7 +189,7 @@ int main(int argc, char* argv[]) {
         next_step();
 
         Core ie;
-        if (FLAGS_d.find("CPU") != std::string::npos && !FLAGS_l.empty()) {
+        if (FLAGS_l.empty()) {
```

Comment:
I don't understand this change. Maybe it should be
```suggestion
        if (!FLAGS_l.empty()) {
```

Reply:
Hi, sorry it's my mistake.